gulpfile.js

```diff
@@ -56,6 +56,18 @@ function watch() {
   gulp.watch(copyFileSources, copy);
 }
 
+var typedoc = require('gulp-typedoc');
+gulp.task('typedoc', function() {
```

Comment:
1. Prefer the newer `exports.typedoc = ...` syntax to `gulp.task()` calls.
2. In cases where you're defining an anonymous function, prefer arrow syntax: `() => { /* ... */ }`.
3. I would prefer to see a `function typedoc` declared in the file before exporting; this will make it easier to add as part of the build pipeline.